bin/pyspark

```diff
@@ -61,7 +61,12 @@ export PYTHONPATH="${SPARK_HOME}/python/lib/py4j-0.10.4-src.zip:$PYTHONPATH"
 
 # Load the PySpark shell.py script when ./pyspark is used interactively:
 export OLD_PYTHONSTARTUP="$PYTHONSTARTUP"
-export PYTHONSTARTUP="${SPARK_HOME}/python/pyspark/shell.py"
+# Check if we are pip installed or otherwise
```

Comment:
Why is this necessary? Could this be a single code path? That would be better if possible.